src/styles/blocks/styles.scss

```diff
@@ -0,0 +1,22 @@
+.page {
```

Comment:
file should have the same name as BEM block

Reply:
Yeh, I also have a class"page" for whole page) and than after header "page__cards-container"

Reply:
isn't it a mixed BEM structure?

Reply:
Yeap, so you should create the separate file for the BEM block